corehq/util/__init__.py

```diff
@@ -1,5 +1,6 @@
 import re
 
+from .couch import get_document_or_404
```

Comment:
True.  I think that's a good direction to head in though.  If we decide to stick with full-path imports, I'd be fine with following that convention.